src/page/project/environment.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ProjectEnvironment from '../../component/project/ProjectEnvironment/ProjectEnvironment';
+import PropTypes from 'prop-types';
+
+const render = ({ history }) => <ProjectEnvironment history={history} />;
```

Comment:
Is this needed? We could use useHistory inside of the component to access history and avoid the page abstraction.

Reply:
not needed